source/support.tex

```diff
@@ -635,6 +635,8 @@
 #define @\defnlibxname{cpp_lib_format_ranges}@                     202207L // also in \libheader{format}
 #define @\defnlibxname{cpp_lib_formatters}@                        202302L // also in \libheader{stacktrace}, \libheader{thread}
 #define @\defnlibxname{cpp_lib_forward_like}@                      202207L // freestanding, also in \libheader{utility}
+#define @\defnlibxname{cpp_lib_freestanding_algorithm}@            202311L // freestanding, also in \libheader{algorithm}
```

Comment:
@jensmaurer, @jwakely Please note that I'm reconciling this with Motion 2 by adding `<numeric>` here, too, for the saturation arithmetic.

Reply:
@jwakely OK, we should make a decision here. Adding a new macro for `numeric` seems reasonable, too, shall I go ahead with that?

Reply:
Yes, I think so. I've pinged Barry and Ben on the PR for that change, for extra eyes.